global-cli/index.js

```diff
@@ -166,3 +169,13 @@ function checkNodeVersion() {
     process.exit(1);
   }
 }
+
+function isGitHubBoilerplate(root) {
+  var validFiles = [
+    '.DS_Store', 'Thumbs.db', '.git', '.gitignore', 'README.md', 'LICENSE'
```

Comment:
Please open another issue to discuss this. Comments on old issues aren't very actionable.